tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable global-require */
```

Comment:
- [ ] Kindly remove every eslint disable rule and instead fix the issue in the code.